src/components/Routes/Routes.tsx

```diff
@@ -16,109 +19,64 @@ import SystemsIndex from '../../pages/SystemIndex/SystemIndex'
 import Login from '../../pages/Login'
 import GardenAdminView from '../../pages/GardenAdminView/GardenAdminView'
 import RequireAuth from './RequireAuth'
+import { ServerConfigContainer } from '../../containers/ConfigContainer'
 
 const Routes = () => {
+  const { getConfig } = ServerConfigContainer.useContainer()
+  const [ authIsEnabled, setAuthIsEnabled ] = useState<boolean | undefined>()
+
+  useEffect(() => {
+    const fetchData = async () => {
+      const data = await getConfig()
+  
+      setAuthIsEnabled(data?.auth_enabled);
+    }
+ 
+    fetchData();
+  }, [])
```

Comment:
I'm happy with how the code in this PR operates and will merge without changes.

However, I'd like to eventually address why this dependency array needs to be empty to avoid an infinite loop.

I think I've tracked it back to the axios instance being improperly or incorrectly updated in the token code (regardless of whether authorization is enabled). But fixing that is a whole other big ugly can of worms and will therefore fall under its own issue.